### PR TITLE
GPII-3641 GPII-3861 GPII-3871 GPII-3856 GPII-3857 Various fixes

### PR DESCRIPTION
**Pull request by mrtyler**
This PR is on top of #368. First new commit is 057e2e4. Note that hashes don't line up (but commit messages do) because I rebased this branch because Github didn't like the merge I did as my first try.

This needs a little more testing and ticket-related bookkeeping, but so far these changes all seem to work.

* Replace busybox with alpine for couchdb init container
* Update couchdb-prometheus-exporter's prometheus-to-sd to the latest, 0.5.1
* Update nginx-ingress to the latest, 0.21.1 (I plan to update the chart as well while I'm doing this)
* Re-run port-forward each time when checking membership for couchdb.finish_cluster

### CODE REVIEW

File: shared/versions.yml [hunk 1]
```diff
@@ -10,110 +10,110 @@ cert_manager:
     repository: quay.io/jetstack/cert-manager-controller
     tag: v0.6.1
   generated:
-    image: gcr.io/gpii-common-prd/quay.io/jetstack/cert-manager-controller
+    repository: gcr.io/gpii2test-common-stg/quay.io/jetstack/cert-manager-controller
```
Comment (mrtyler) [inline shared/versions.yml]: This value is incorrect (except for my testing), but version-updater will fix it once everything is merged.

**Comment by natarajaya** [comment]
LGTM

**Comment by mrtyler** [comment]
Chart is updated. I also bumped couchdb-statefulset-assembler to the latest, for fun.

I was going to chop this branch into a bunch of PRs for each ticket, but it's kind of a pain for little benefit so I plan to merge this as-is (assuming it passes a few more tests).

File: shared/versions.yml [hunk 2]
```diff
@@ -6,86 +6,114 @@
...
   generated:
-    image: gcr.io/gpii-common-prd/elsdoerfer/k8s-snapshots
+    repository: gcr.io/gpii2test-common-stg/elsdoerfer/k8s-snapshots
     sha: sha256:c787dc6fa1812c97ac96f46b4ff6dc1485e06c458839d4aa129aaf62201309e9
     tag: v2.0
 locust:
-  upstream_image: gpii/locust:0.9.0-gpii.4
+  upstream:
+    repository: gpii/locust
+    tag: 0.9.0-gpii.4
```
[the remark this answers is not in the document]
Comment (mrtyler) [reply shared/versions.yml]: Thanks :). I fixed this in https://github.com/gpii-ops/gpii-infra/pull/368.

**Comment by mrtyler** [comment]
Closing in favor of https://github.com/gpii-ops/gpii-infra/pull/373.

**Pull request closed without merging**